ads/google/doubleclick.js

```diff
@@ -65,15 +65,30 @@ export function doubleclick(global, data) {
     transform: 'translate(-50%, -50%)',
   });
 
-  if (data.useSameDomainRenderingUntilDeprecated != undefined ||
-      data.multiSize) {
-    doubleClickWithGpt(global, data, GladeExperiment.GLADE_OPT_OUT);
+  let url = 'https://www.googletagservices.com/tag/js/';
+  let sf = false;
+  const expIds = data['experimentId'];
```

Comment:
const fileExperimentConfig = {
   21060540: 'gpt_sf_a.js',
   21060541: 'gpt_sf_b.js',
}
// Note that reduce will return the first item that matches but it is expected that only one of the experiment ids will be present.
const expFilename = data['experimentId'] && data['experimentId'].split(',').reduce(expId => fileExperimentConfig[expId]);
const url = `https://www.googletagservices.com/tag/js/${expFilename || 'gpt.js'}`;
if (data.useSameDomainRenderingUntilDeprecated != undefined || data.multiSize || expFilename) {
   doubleClickWithGpt(global, data, GladeExperiment.GLADE_OPT_OUT, url);
}